src/server/actions/mailMerge.ts

```diff
@@ -14,6 +18,18 @@ export async function getAllMailMerge(): Promise<MailMergeResponse[]> {
   }
 }
 
+export async function getMailMergeByType(
+  type: string
+): Promise<MailMergeResponse> {
+  await dbConnect();
+
+  const response = await MailMergeSchema.findOne({ type: type });
+
+  if (!response) throw Error('Document not found');
```

Comment:
This wouldn't be an error, a mail merge not existing for a given type is a valid database state. A find by id not getting a response would be an error